Dockerfile

```diff
@@ -11,7 +11,7 @@ RUN apt-get update && apt-get install -y \
     && rm -rf /var/lib/apt/lists/*
 
 # Install Rust and the cargo-ament-build plugin
-RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- --default-toolchain 1.59.0 -y
```

Comment:
Instead of leaving out the version of the toolchain, I'd update it to the latest one so we know for sure which version is the one in the toolchain. Could you update it to 1.62.0? Thanks.